recipe/bld.bat

```diff
@@ -4,13 +4,7 @@ set SETUPTOOLS_SCM_PRETEND_VERSION_FOR_CIL=%PKG_VERSION%
 if not "%GIT_DESCRIBE_NUMBER%"=="0" (
     set SETUPTOOLS_SCM_PRETEND_VERSION_FOR_CIL=%PKG_VERSION%.dev%GIT_DESCRIBE_NUMBER%+%GIT_DESCRIBE_HASH%
 )
-
 :: -G "Visual Studio 16 2019" specifies the the generator
 :: -T v142 specifies the toolset
-
-cmake -S "%RECIPE_DIR%\.." -B "%SRC_DIR%\build_framework" -G "Visual Studio 16 2019" -T "v142" -DCONDA_BUILD=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo -DLIBRARY_LIB=%CONDA_PREFIX%\lib -DLIBRARY_INC=%CONDA_PREFIX% -DCMAKE_INSTALL_PREFIX=%PREFIX% -DPython_EXECUTABLE=%CONDA_PREFIX%\python
-
-if errorlevel 1 exit 1
-
-cmake --build "%SRC_DIR%\build_framework" --target install --config RelWithDebInfo
+pip install . --no-deps -Ccmake.args="-G Visual Studio 16 2019 -T v142"

```

Comment:
why is it necessary to specify these arguments? https://conda-forge.org/docs/maintainer/knowledge_base/#simple-cmake-based-bldbat doesn't do that.

Reply:
@gfardell currently manually builds & uploads CIL windows conda packages on her machine and she wants/needs this.

After proper CI/CD (#1918 etc.) it will/should disappear.